front-end/key.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Since the name of this file is "key" and it has a private key block in it, I want to confirm this is Github shareable. The PEM extension usually refer to the certificate file and not the key, but since the name is "key" I though I would check anyway.

Reply:
This certificate is for local developpement only. When the site goes to production another mecanism inserts valid https certificates. The use of having this in the repo si to simplify the needs for manual changes before beeing able to run the whole application using docker compose. Maybe in a next step we could include a certificate generator in the docker image, but that still doesn't correct the probleme when debuggin the application directly in the host os of the developper.